tests/integration/test_backup_log/test.py

```diff
@@ -0,0 +1,58 @@
+import pytest
+from helpers.cluster import ClickHouseCluster
+from helpers.test_tools import TSV
+
+
+cluster = ClickHouseCluster(__file__)
+
+instance = cluster.add_instance(
+    "instance",
+    main_configs=["configs/config.xml", "configs/config.d/backups.xml"],
+    stay_alive=True,
```

Comment:
`stay_alive` is not necessary, your test doesn't restart the server

Reply:
Actually, it restarts the server (`instance.restart_clickhouse()` on line 50), just to ensure that the `MergeTree`-based system log stays persistent afterwards. 
Without `stay_alive` parameter set to `True`, the `restart_clickhouse` method in `cluster.py` throws an exception `clickhouse can be stopped only with stay_alive=True instance`.